samples/Samples.MultiDomainHost.Runner/Program.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.IO;
+using System.Net;
+using System.Security;
+using System.Security.Permissions;
+using System.Threading;
+
+namespace Samples.MultiDomainHost.Runner
+{
+    public class Program
+    {
+        [LoaderOptimization(LoaderOptimization.MultiDomainHost)]
+        static int Main(string[] args)
+        {
+            Console.WriteLine("Loading System.dll early in the process");
+
+            using (var webClient = new WebClient())
+            {
+            }
+
+            try
+            {
+                // First load the application that does not have bindingRedirect policies
+                // This will instrument at least one domain-neutral assembly with a
+                // domain-neutral version of Datadog.Trace.ClrProfiler.Managed.dll
+                CreateAndRunAppDomain("Samples.MultiDomainHost.App.FrameworkHttpNoRedirects");
+
+                // Next load an application that does the same thing, still does not have
+                // bindingRedirect policies, but it uses the System.Net.Http NuGet package
+                // instead of the built-in version.
+                // TBD if this breaks in the non-GAC scenario
```

Comment:
Is this still a TBD?

Reply:
Good find, nope this is not TBD. I'll go ahead and remove this comment. I should probably search for "TBD" and "TODO" in this PR...

Reply:
Fixed in 3ec86bfc6d5dd4c2137aa4a825fff13c6bc0b863